lib/smartsheet/response_fixtures.ex

```diff
@@ -329,6 +329,245 @@ defmodule Smartsheet.MockClient.ResponseFixtures do
+          "resultCode" => 0,
+          "result" => %{
+            "id" => 8_336_906_762_119_044,
+            "name" => "test 1",
+            "scope" => "sheet",
+            "scopeObjectId" => 3_944_882_986_346_372,
+            "subscope" => %{
+              "columnIds" => [
+                3_882_534_776_924_036
+              ]
+            },
+            "events" => [
+              "*.*"
+            ],
+            "callbackUrl" => "https =>//0205c71fedd0.ngrok.io/api/v1/verify_webhook",
+            "sharedSecret" => "6gab00fcynp9vh6p29vfi3qne7",
+            "enabled" => false,
+            "status" => "NEW_NOT_VERIFIED",
+            "version" => 1,
+            "createdAt" => "2020-10-06T15:01:14Z",
+            "modifiedAt" => "2020-10-06T15:01:14Z"
+          }
+        },
+        result_code: 0,
+        version: 4
+      },
+      headers: [
+        {"Date", "Tue, 08 Sep 2020 16:31:31 GMT"},
+        {"Content-Type", "application/json;charset=UTF-8"},
+        {"Content-Length", "358"},
+        {"Connection", "keep-alive"},
+        {"Cache-Control", "no-cache, no-store, must-revalidate"},
+        {"Pragma", "no-cache"},
+        {"Expires", "0"},
+        {"Vary", "Accept-Encoding"}
+      ],
+      request: %HTTPoison.Request{
+        body: "{ \"name\": \"test 1\" } ",
+        headers: [
+          Authorization: "Bearer 123",
+          "Content-Type": "application/json"
+        ],
+        method: :post,
+        options: [],
+        params: %{},
+        url: "https://api.smartsheet.com/2.0/webhooks"
+      },
+      request_url: "https://api.smartsheet.com/2.0/webhooks",
+      status_code: 200
+    })
+  end
+
+  def update_webhook_failure() do
+    format_response(%HTTPoison.Response{
+      body: %{
+        "errorCode" => 1006,
+        "message" => "Not Found",
+        "refId" => "19a3k3trvdizp"
+      },
+      headers: [
+        {"Date", "Tue, 08 Sep 2020 16:33:15 GMT"},
+        {"Content-Type", "application/json;charset=UTF-8"},
+        {"Content-Length", "180"},
+        {"Connection", "keep-alive"},
+        {"Vary", "Accept-Encoding"}
+      ],
+      request: %HTTPoison.Request{
+        body: "{ \"enabled\": \"true\" } ",
+        headers: [
+          Authorization: "Bearer 123",
+          "Content-Type": "application/json"
+        ],
+        method: :put,
+        options: [],
+        params: %{},
+        url: "https://api.smartsheet.com/2.0/webhooks/1407028569630596"
+      },
+      request_url: "https://api.smartsheet.com/2.0/webhooks/1407028569630596",
+      status_code: 400
+    })
+  end
+
+  def update_webhook_success() do
+    format_response(%HTTPoison.Response{
+      body: %{
+        message: "SUCCESS",
+        result: %{
+          "message" => "SUCCESS",
+          "resultCode" => 0,
+          "result" => %{
+            "id" => 8_336_906_762_119_044,
+            "name" => "test 1",
+            "scope" => "sheet",
+            "scopeObjectId" => 3_944_882_986_346_372,
```

Comment:
this PR is making me think perhaps i should write some docs about this library.

the naked response from HTTPoison will definitely be camelCased strings, but when you go through the HTTPClient, it will parse the body and make sure that the keys are snake_cased and atomized. i think these fixtures should reflect that, like the other fixtures. sorry! i shoulda made that clear somewhere / set up some kind of process for this